src/overrides.ts

```diff
@@ -70,6 +71,8 @@ class DefaultOverrides {
       [PokeballType.MASTER_BALL]: 0,
     },
   };
+  /** Forces an item to be UNLOCKED */
+  readonly UNLOCK_OVERRIDE: Unlockables[] = [];
```

Comment:
Please be more specific here

```suggestion
  readonly ITEM_UNLOCK_OVERRIDE: Unlockables[] = [];
```